### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I implemented the following:

- Added styling to match the project's Zeplin's design.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,18 +1,170 @@
...
+.menu {
   list-style-type: none;
+  background-color: #fff;
 }
 
-nav > ul > li {
+.menu > li {
   display: inline-block;
   padding-right: 20px;
+  vertical-align: middle;
```
Comment (afanimartin) [inline src/App.css:29]: This styling seems not to be working. The nav items are not vertically aligned to the middle. Please take a look at the bug and make changes accordingly

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @themonster2015, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (themonster2015) [reply src/App.css:29]: thanks. I just fixed it.

**Review by wandji20 — APPROVED** [review]
## Hi @themonster2015  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 
Your project has been approved, however, I have left some comments to help improve your project

## Congratulations :+1:  :fireworks: 
## Optional
- [ optional ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .eslintrc.json etc).
- [ optional ] It will be nice to give some more space between the progress status and chapter summary so they look more separated.
- [ optional ] Centralizing the progress status along the cross axis for each book will be nice. (compare screenshots)
Screenshot from 2021-10-01 11-13-02

screen

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**